wiki/handler-patterns.md

```
@@ -128,3 +128,58 @@ For **full control** over legacy error formats, catch domain errors in the handl
+        Meta: map[string]any{
+            "total":   total,
+            "limit":   req.Limit,
+            "offset":  req.Offset,
+            "hasMore": req.Offset+len(users) < total,
+        },
+    }, nil
+}
+```
+
+Produces:
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)